_data/packages.yml

```diff
@@ -758,7 +795,7 @@
       github_username: acuschwarze
     - name: Thomas Robiglio
       github_username: thomasrobiglio
-    - name: Alice Patania
+    - name: alice patania
```

Comment:
This makes sense. it did that because the name is lowercase in the contributors file. 

https://github.com/pyOpenSci/pyopensci.github.io/blob/main/_data/contributors.yml#L4850

in this case as well, Alice has their name lower case on GitHub

https://github.com/alpatania

So we can update this - we just want to do it in contributors.yml :) it's really good to walk through this as these issues look super random. some of them are bugs others are just how the workflow is setup and because it's not documented well enough.